modules/gapi/test/cpu/gapi_imgproc_tests_fluid.cpp

```diff
@@ -106,7 +106,7 @@ INSTANTIATE_TEST_CASE_P(gaussBlurTestFluid, GaussianBlurTest,
                                 Values(-1),
                                 Values(IMGPROC_FLUID),
                                 Values(ToleranceFilter(1e-3f, 0.01).to_compare_obj()),
-                                Values(3))); // add kernel size=5 when implementation is ready
+                                Values(3, 5))); // add kernel size=5 when implementation is ready
```

Comment:
As you can see here, I also added numeral 5 to instantiation's parameters of accuracy tests for fluid's GaussianBlur. So now all accuracy tests run not only for 3x3 but also for 5x5 kernel size according to our test API.